src/main/resources/mpack.json

```diff
@@ -0,0 +1,31 @@
+{
+  "type": "full-release",
+  "name": "cdap-ambari-mpack",
```

Comment:
this should probably also be consistent with the naming.  `cdap-ambari-service-mpack` vs `cdap-ambari-mpack` vs `cdap-mpack`